### DISCUSSION TITLE
Firebase messaging - SSL Certificates

### DISCUSSION THREAD

**Discussion by JeremyBradshaw7** [invertase/react-native-firebase#6451] [Q&A] [answered]
It's documented [here](https://rnfirebase.io/messaging/usage/ios-setup#2-registering-an-app-identifier) that you have to turn on the `Push Notifications` capability under Apple Developer:

![image](https://user-images.githubusercontent.com/36155348/183067726-d53a5968-d27f-4147-b625-187ca02b1f9a.png)

However it's lacking any documentation on what (if anything) you have to do under the Configure button:

![image](https://user-images.githubusercontent.com/36155348/183067609-ad6d3228-096f-4778-8e69-d79c1675befa.png)

Are SSL Certificates necessary? We didn't set any up for testing under our development app id and push notifications still worked. Will we hit an issue in production without this configuration? The documentation is unclear on this point.

**Comment by mikehardy** [comment]
There is no documentation there because there is nothing to do there - you do not need to set up SSL certificates. It's been a while since you asked (apologies) - I imagine you deployed to production and it worked, yes?

**Comment by JeremyBradshaw7** [comment, marked as answer]
Yes, we took a flier and deployed it to production and it just worked without SSL certificates. Thanks.

**Reply by mikehardy** [reply]
Okay great to hear - this may just be confirmation bias but to the best of my knowledge using FCM means they manage the connection between APNs servers and themselves so they handle that part. If you were to *directly* send messages to APNs then you'd need these certs, but that takes most of the "fun" out of it because then you are managing messaging differently on two platforms (vs unified FCM)

Good luck with your project!
